src/lib/csvParser.js

```diff
@@ -0,0 +1,116 @@
+};
+
+// Replace later if needed- looking for a standard way to
+const precinctIdentifier = rowObject => {
+  const { precinctFull } = rowObject;
+  return `${precinctFull}`;
+};
+
+const rowToData = rowObject => {
+  const key = precinctIdentifier(rowObject);
+  const {
+    // Issues - keep up to date w/ csv model
+    viableLoss,
+    nonviableNoRealign,
+    alphaShift,
+    hasAlphaShift,
+    moreFinalVotes,
+    fewerFinalVotes,
+    delCountsDiff,
+    extraDelGiven,
+    // End Issues
+
+    ...cleanData
+  } = rowObject;
+
+  const issues = parseIssues({
+    viableLoss,
+    nonviableNoRealign,
+    alphaShift,
+    hasAlphaShift,
+    moreFinalVotes,
+    fewerFinalVotes,
+    delCountsDiff,
+    extraDelGiven
+  });
+
+  return {
+    key,
+    issues,
+    ...cleanData
+  };
+};
+
+const parseIssues = ({
+  viableLoss,
+  nonviableNoRealign,
+  alphaShift,
+  hasAlphaShift,
+  moreFinalVotes,
+  fewerFinalVotes,
+  delCountsDiff,
+  extraDelGiven
+}) => {
+  const issues = {
+    alphaShift: hasAlphaShift && {
+      candidate: alphaShift
+    },
+    viableLoss: coerceBool(viableLoss),
+    nonviableNoRealign: coerceBool(nonviableNoRealign),
+    moreFinalVotes: coerceBool(moreFinalVotes),
+    fewerFinalVotes: coerceBool(fewerFinalVotes),
+    delCountsDiff: delCountsDiff || 0,
+    extraDelGiven: coerceBool(extraDelGiven)
+  };
+  // console.log(issues);
+  return issues;
+};
+
+const coerceBool = val => !!val && val.toUpperCase() === "TRUE";
+
+/*
+# A tibble: 19 x 2
+   colnames             description                                                                                                                
+   <chr>                <chr>                                                                                                                      
+ 1 county               ""                                                                                                                         
+ 2 precinct             ""                                                                                                                         
+ 3 candidate            ""                                                                                                                         
+ 4 precinct_delegates   # of delegates to be given by precinct                                                                                     
+ 5 align1               # of votes for candidate in 1st alignment                                                                                  
+ 6 alignfinal           # of votes for candidate in final alignment                                                                                
+ 7 final_del            our calculated # of delegates earned                                                                                       
+ 8 reported_del         the reported # of delegates earned                                                                                         
```

Comment:
For reference - i'm camelizing keys when parsing the CSV so we'll never have the snake case to deal with in JS.